_maps/map_files/LV624/LV624.dmm

```diff
@@ -329,12 +329,12 @@
 /area/lv624/ground/sand6)
 "ahH" = (
 /obj/structure/morgue/sarcophagus,
-/obj/item/weapon/combat_knife{
-	desc = "A curved blade made of a strange material. It looks both old and very sharp.";
-	force = 30;
-	icon_state = "unathiknife";
+/obj/item/weapon/katana{
+	desc = "Woah! Holy shit! A glowing alien sword! How fucking cool is that, and it's super sharp!";
+	force = 85;
+	icon_state = "blade";
 	name = "\improper alien blade";
-	throwforce = 26
+	throwforce = 45
```

Comment:
Yea absolutely no remove all these varedits

Reply:
It's probably for the best to remove it (what with varedits being bad), I'll just stick a standard sword of some type in.